src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -146,14 +146,14 @@ public partial class NetworkStream : System.IO.Stream
         public override int Read(byte[] buffer, int offset, int count) { throw null; }
         public override int Read(System.Span<byte> buffer) { throw null; }
         public override System.Threading.Tasks.Task<int> ReadAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
-        public override System.Threading.Tasks.ValueTask<int> ReadAsync(System.Memory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override System.Threading.Tasks.ValueTask<int> ReadAsync(System.Memory<byte> buffer, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
I think we want to keep the default for the cancellation token here, and add it to the sources instead.

Reply:
`Stream.ReadAsync` that this is overriding has the default for the cancellation token as well.

Reply:
ok. My primary goal here to get to point where GenerateReferenceAssemblySource just work.